starter-code/app.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express");
+const app = express();
+const port = 3000;
+
+app.use(express.static('public'));
+
+
+app.get('/',(req,res) => {
+    res.sendFile(__dirname + '/views/index.html');
+})
+
+app.get('/about.html',(req,res) => {
+    res.sendFile(__dirname + '/views/about.html');
+})
+
+app.get('/gallery.html', (req, res) => {
+    res.sendFile(__dirname + '/views/gallery.html');
+})
+
+
+app.listen(port,() => {
+    console.log('Listening port ${port}');
```

Comment:
Template literals are enclosed by the back-tick (` `)  character instead of double or single quotes.